cmd/importer/importer.go

```diff
@@ -40,14 +41,14 @@ func main() {
 	sec := ParseEnvVar(common.IMPORTER_SECRET_KEY, false)
 	fn := filepath.Base(ep.Path)
 	if !image.IsSupporedFileType(fn) {
-		glog.Errorf("main: unsupported source file %q. Supported types: %v\n", fn, image.SupportedFileExtensions)
+		glog.Error(Errf("unsupported source file %q. Supported types: %v\n", fn, image.SupportedFileExtensions))
```

Comment:
why is a stack trace needed here since the trace will just be _main_ (since `IsSupporedFileType` returns a bool)?

Reply:
Any instance of `Err` or `Errf` is just for experimentation right now.